### PR TITLE
Update README about flags on cluster logs

### PR DESCRIPTION
**Pull request by chxchx**
**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/e2e/README.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ Tests could be built and drove manually as a single test or automatically detect
 * `--verbose` debug level noise from proxies
 * `--istioctl_url` the location of an `istioctl` binary
 * `--skip_cleanup` if skip cleanup steps
+* `--log_provider` where cluster logs are hosted, only support stackdriver for now
```
Comment (gyliu513) [inline tests/e2e/README.md]: how about s/stackdriver/`stackdriver`

Comment (chxchx) [inline tests/e2e/README.md]: Done

**Pull request merged**